storybook/components/DosAndDonts/DoesAndDonts.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { HTMLAttributes } from "react"
+import { Canvas } from "@storybook/blocks"
+import classnames from "classnames"
+import { CheckIcon } from "~components/Icon"
+import { SbContent } from "../SbContent"
+
+export const DosAndDonts = ({
+  className,
+  children,
+  ...otherProps
+}: HTMLAttributes<HTMLDivElement>): JSX.Element => (
+  <div
+    className={classnames(className, "grid grid-cols-1 gap-16 lg:grid-cols-2")}
+    {...otherProps}
+  >
+    {children}
+  </div>
+)
+
+DosAndDonts.displayName = "DosAndDonts"
```

Comment:
nit: setting displayName manually is not needed anymore with modern react & tools. It detects that automatically.

Reply:
Ah cool, good to know. We should probs update our plop templates going forward and drop this pattern from any rebuilds in the versioned releases